components/dashboard/conf/Caddyfile

```diff
@@ -7,29 +7,38 @@
 	encode zstd gzip
 }
 
-(caching) {
-	header Cache-Control "public, max-age=604800, must-revalidate"
-}
-
-(default) {
-	header -Server
-}
-
 (discard_log) {
 	log {
 		output discard
 	}
 }
 
 :80 {
-	import caching
+	@static {
+		path /static/*
+	}
+
+	@notstatic {
+		not path /static/*
+	}
+
 	import compression
-	import default
 	import discard_log
 
 	redir /.well-known/security.txt https://www.gitpod.io/.well-known/security.txt permanent
 	redir /environment-variables /settings/ permanent
 
+	handle @static {
+		header Cache-Control "public, max-age=31536000"
+	}
+
+	handle @notstatic {
+		header Cache-Control "no-cache, no-transform, must-revalidate"
```

Comment:
I'd expect at least a short comment about the two types of content and about the intent and trade-offs regarding the two caching strategies.